covid/extract.py

```diff
@@ -183,22 +90,24 @@ def extract_cdc_ili_data():
     return df
 
 
-def extract_cdc_beds_current_data():
-    inpatient_bed_df = extract_cdc_inpatient_beds()
-    icu_bed_df = extract_cdc_icu_beds()
-    hospitals_reporting_df = extract_cdc_facilities_reporting()
-    cdc_df = pd.concat([inpatient_bed_df, icu_bed_df, hospitals_reporting_df], axis=1)
-    cdc_df = cdc_df.loc[:, ~cdc_df.columns.duplicated()]
-    return cdc_df
+def extract_hhs_icu_data():
+    hhs_pro_pub_url = "https://opendata.arcgis.com/datasets/f527cf1835294eb280a4e5ee3c5c209c_0.geojson"
+    geo_j = requests.get(hhs_pro_pub_url).json()
+    df = pd.DataFrame([feat['properties'] for feat in geo_j['features']])
+    df.drop('OBJECTID', axis=1, inplace=True)
+    df.rename({'state_name': STATE_FIELD, 'last_updated': DATE_SOURCE_FIELD}, axis=1, inplace=True)
+    df[DATE_SOURCE_FIELD] = df[DATE_SOURCE_FIELD].str[:10]
+    df.set_index(STATE_FIELD, inplace=True)
+    return df
 
 
-def extract_cdc_beds_historical_data(credentials):
-    cdc_historical_df = gspread2df.download(
-        CATEGORY_3_DATA_GOOGLE_SHEET_KEY,
-        CATEGORY_3_HISTORICAL_DATA_TAB,
-        credentials=credentials,
-        col_names=True,
-    )
-    cdc_historical_df = cdc_historical_df.set_index(STATE_FIELD)
+# def extract_cdc_beds_historical_data(credentials):
```

Comment:
@BriceWolfgang 3 options for regenerating the historical data:
(1) Keep the existing sheet and tab the same, either rename the columns in the sheet to match the new column titles (or revert the column title change)
(2) Create an entirely new sheet to track the category 3 data to show we're using new source now
(3) Add a new tab to the existing sheet with new column labels, etc. and refer to that one as our historical source of truth

@lucasmbrown any thoughts?